test/test_binary_ufuncs.py

```diff
@@ -1183,6 +1184,34 @@ def test_binary_ops_with_scalars(self, device):
                         self.assertEqual(expected, python_op(first, second))
                         self.assertEqual(expected, torch_op(first, second))
 
+    # Tests that the function and its (array-accepting) reference produce the same
+    #   values on a range of tensors, including empty tensors, scalar tensors,
+    #   1D tensors and a large 2D tensor
+    @onlyOnCPUAndCUDA
+    @ops(binary_ufuncs)
+    def test_reference_numerics(self, device, dtype, op):
+        if dtype == torch.bfloat16:
+            np_dtype = np.float16
+
+            def to_numpy(t):
+                return np.array(t.tolist(), dtype=np_dtype)
+
+        else:
+            np_dtype = torch_to_numpy_dtype_dict[dtype]
+
+            def to_numpy(t):
+                return np.array(t.cpu(), dtype=np_dtype)
```

Comment:
If we need this pattern in multiple tests, we should factor it out.

Reply:
See above comment